cypress/e2e/polls/polls.spec.ts

```diff
@@ -70,8 +68,16 @@ describe("Polls", () => {
             cy.get('input[type="radio"]')
                 .invoke("attr", "value")
                 .then((optionId) => {
-                    const pollVote = PollResponseEvent.from([optionId], pollId).serialize();
-                    bot.sendEvent(roomId, pollVote.type, pollVote.content);
+                    // We can't use the js-sdk types for this stuff directly, so manually construct the event.
+                    bot.sendEvent(roomId, "org.matrix.msc3381.poll.response", {
```

Comment:
Don't we have an event type for this?

Reply:
oh we do, but if we import it then the test explodes in a ball of fire. We seem to manually cast everywhere else and avoid importing types, so mirroring that behaviour here.

Reply:
You can argue it's a good thing that your tests construct things from scratch rather than importing the the implementation from the main code; in general avoiding code reuse between implementation and tests can help avoid just making the same error twice.

Reply:
also that: I generally prefer to hardcode strings like event types to force tests to break when the unstable/stable relationship is improperly handled.